src/components/taskDetails/index.tsx

```diff
@@ -430,11 +432,16 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
                                         taskDetailsData?.startedOn
                                     )}
                                 />
-
                                 <Details
                                     detailType={'Ends On'}
                                     value={getEndsOn(taskDetailsData?.endsOn)}
+                                    icon={<FaReceipt color="green" />}
+                                    icon_url={`${TASK_EXTENSION_REQUEST_URL}?taskId=${taskDetailsData?.id}`}
```

Comment:
- should we store the url in a variable above the return statement ? It will be helpful if we want to use it anywhere else in the component and act as a single source of truth.